doc/Developers-Intro.md

```diff
@@ -42,29 +42,43 @@ If you have seen Friendica you probably have ideas to improve it, haven't you?
 * Make plans for a better Friendica interface design and share them with us.
 * Tell us if you are able to realize your ideas or what kind of help you need.
 We can't promise we have the right skills in the group but we'll try.
-* Choose a thing to start with, e.g. work on the icon set of your favourite theme
+* Choose a thing to start with, e.g. work on the icon set of your favorite theme
 
 Programming
 ---
 
+###Coding standards
+
+For the sake of consistency between contribution and general code readability, Friendica follows the widespread [PSR-2 coding standards](http://www.php-fig.org/psr/psr-2/). Don't worry, you don't have to know by heart the PSR-2 coding standards. Before comitting your work, you can use the PHP Coding Standard Fixer located in the `util` folder this way:
```

Comment:
We are using the rule of one-sentence-per-line for the docs. Could you change this.